packages/block-library/src/navigation-link/editor.scss

```diff
@@ -33,13 +39,13 @@
 	}
 }
 
-// Separator
+// Separator. @todo: this appears unused.
 .wp-block-navigation-link__separator {
 	margin: $grid-unit-10 0 $grid-unit-10;
 	border-top: $border-width solid $gray-300;
 }
 
-// Popover styles
+// Popover styles. @todo: these all appear unused.
```

Comment:
If this is also unused (which it seems to be), it seems like a possible candidate for removal too.